docs/content/documentation/systems/Materials/tensor_mechanics/GeneralizedMaxwellModel.md

```diff
@@ -1,68 +1,38 @@
-# GeneralizedMaxwellModel
+# Generalized Maxwell Model
+!syntax description /Materials/GeneralizedMaxwellModel
 
-This material represents a generalized Maxwell model, that is, a material composed of $N$ Maxwell units assembled in series.
+## Description
+The `GeneralizedMaxwellModel` class represents a generalized Maxwell model, that is, a material composed of $N$ Maxwell units assembled in series.
 
 ### Constitutive Equations
 The material obeys to the following constitutive equation:
-
-$$
-\sigma_{ij} = C_{ijkl} \epsilon^{mech}_{kl} - \sum\limits_{n=1}^{N} C^{n}_{ijkl} \alpha^{n}_{kl} 
-$$
+\begin{equation}
+\sigma_{ij} = C_{ijkl} \epsilon^{mech}_{kl} - \sum\limits_{n=1}^{N} C^{n}_{ijkl} \alpha^{n}_{kl}
+\end{equation}
 
 The $\alpha^{n}$ are the internal strains associated to each Maxwell unit, and $C^{n}$ the stiffness of the corresponding spring (a fourth-order tensor, identical in symmetry and dimensions to a standard elasticity tensor). The $\alpha^{n}$ obey the following time-dependent differential equation:
-
-$$
+\begin{equation}
 \forall n \in [1, N]: \epsilon^{mech} - \alpha^{n} = \eta^{n} \dot{\alpha}^{n}
-$$
-
+\end{equation}
 With $\eta^{n}$ is the viscosity of the associated dashpot (a scalar with the dimension of time).
 
-### Internal Time-Stepping Scheme
-
-The constitutive equations are solved using a semi-implicit single-step first-order finite difference scheme. The internal strains at time step $t+\Delta t$ are computed from their values at the previous time step $t$:
-
-$$
-\alpha^{n}(t+\Delta t) = \alpha^{n}(t) + \Delta t \left( \theta^n \dot{\alpha}^n (t+\Delta t) + \left( 1 - \theta^n \right) \dot{alpha}^n (t)  \right)
-$$
-
-$\theta$ is a scalar between 0 (fully explicit) and 1 (fully implicit) that controls the time-stepping scheme (default value: 1). $\theta$ is determined by the "integration_rule" input parameter, which can take one of the following values:
-
-| Rule | Value of $\theta$  | Unconditional convergence |
-| BackwardEuler | 1 | yes |
-| MidPoint | 0.5 | yes |
-| Newmark | user-defined | $\theta \geq 0.5$ |
-| Zienkiewicz | $\frac{1}{1 - e^{-\Delta t / \eta^n}} - \eta^n / \Delta t$ | yes |
+!include /GeneralizedKelvinVoigtModel.md start=Internal Time-Stepping Scheme end=Example Input File Syntax
```

Comment:
I'd prefer that we don't do this kind of thing because someone editing that page might have no idea that part of it could be included elsewhere.  I think it's better to just have a link to that page instead to refer the reader to that material.

Reply:
Okay, I can see that the gratification of deleting duplicated information doesn't outweigh the chance of incorrect documentation in these cases.  I'll put it back to how it was originally.